tempodb/backend/gcs/gcs.go

```diff
@@ -298,7 +279,7 @@ func (rw *readerWriter) readAllWithModTime(ctx context.Context, name string) ([]
 }
 
 func (rw *readerWriter) readRange(ctx context.Context, name string, offset int64, buffer []byte) error {
-	r, err := rw.bucket.Object(name).NewRangeReader(ctx, offset, int64(len(buffer)))
+	r, err := rw.hedgedBucket.Object(name).NewRangeReader(ctx, offset, int64(len(buffer)))
```

Comment:
I wonder if 500ms timeouts on readRange would affect the compactors? Since we pull a few MB at a time